packages/typography/src/elements/Span.tsx

```diff
@@ -10,16 +10,16 @@ import PropTypes from 'prop-types';
 import { StyledFont, StyledIcon } from '../styled';
 
 interface ISpanProps extends HTMLAttributes<HTMLSpanElement> {
-  /** Any valid DOM element for the styled component */
+  /** Updates the element tag */
   tag?: any;
-  /** Render bold font; `undefined` = inherited weight, `true` = bolded weight, `false` = regular weight */
+  /** Applies bold font style */
```

Comment:
I think it might be important to incorporate the use of `undefined` (or default) here.

Something like: `Applies bold font style. Font weight is inherited by default.`

Reply:
Agreed